spec/services/persist_basic_contained_output_file_service_spec.rb

```diff
@@ -13,13 +13,16 @@ class BasicContainerObject < ActiveFedora::Base
   let(:file)              { File.new(file_path)}
   let(:destination_name)  { 'the_derivative_name' }
 
-  context "when file is basic contained (default assumption)" do  # alas, we have to support this as the default because all legacy code (and fedora 3 systems) created basic contained files
```

Comment:
I was assuming this test was how the legacy code uses derivatives.  I didn't want to alter it with the new wrapper.